resources/functionMap.php

```diff
@@ -7272,7 +7272,7 @@
 'mysqli_multi_query' => ['bool', 'link'=>'mysqli', 'query'=>'string'],
 'mysqli_next_result' => ['bool', 'link'=>'mysqli'],
 'mysqli_num_fields' => ['int', 'link'=>'mysqli_result'],
-'mysqli_num_rows' => ['int', 'link'=>'mysqli_result'],
+'mysqli_num_rows' => ['int<-1,max>|numeric-string', 'link'=>'mysqli_result'],
```

Comment:
When it's -1?

Reply:
Good catch. I just copied the type from `mysqli_affected_rows`. It's fixed.